src/test/java/us/ihmc/zed/test/DemoImageCapture.java

```diff
@@ -58,7 +58,7 @@ public static void main(String[] args) {
         init_param.open_timeout_sec(5.0f);
         init_param.async_grab_camera_recovery(false);
         init_param.grab_compute_capping_fps(0);
-        init_param.enable_image_validity_check(false);
+        init_param.enable_image_validity_check(0);
```

Comment:
We'll need this change in open robotics as well, right?

Reply:
Yes, any dependents will need this change upon upgrading